sdk/storage/azure-storage-blob/tests/test_blob_client_async.py

```diff
@@ -144,7 +145,7 @@ def test_create_service_with_sas_credential_url_raises_if_sas_is_in_uri(self, **
     async def test_create_service_with_token(self, **kwargs):
         storage_account_name = kwargs.pop("storage_account_name")
 
-        token_credential = self.generate_fake_token()
+        token_credential = FakeTokenCredential()
```

Comment:
Is there an async fake token? From `devtools_testutils.aio` perhaps? Doesn't really matter here but if there is I guess we should use it.

Applies to any async file that uses a fake credential.

Reply:
Doesn't exists 😄